tests/test_transaction_validation_in_block.py

```diff
@@ -12,7 +12,7 @@ def test_conflicting_transactions_in_block():
     amount = 110
     token = add_token(from_wallet['address'], amount)
 
-    transfer_unilateral(WALLET, from_wallet, 'NWRL', 2 * MIN_NEWRL_FEE)
+    transfer_unilateral(WALLET, from_wallet, 'NWRL', 10 * MIN_NEWRL_FEE)
```

Comment:
@Prannav-k  why is this set to 10?

Reply:
Tests use run updater when env is local/test. In run updater, fee is being deducted once from updater.py ln: 101. and then again on actual fee. And one more time when run updater is executed again . Because of which fee is being deducted 3 times for which I had to increase balance.